### ISSUE TITLE
TypeError: this.setData is not a function in remark-cli

### ISSUE THREAD

**Issue by rickplumeria** [nuxtlabs/remark-mdc#60]
### Version

module: 2.0.0
nuxt: N/A

## Reproduction

```bash
$ npm install remark-cli remark-mdc
$ ./node_modules/.bin/remark --use remark-mdc index.md
index.md
 error Cannot process file
  [cause]:
    TypeError: this.setData is not a function
    at Object.enterAttributes (file:///home/rickplumeria/tmp/remark-mdc-bug/node_modules/remark-mdc/dist/index.mjs:481:8)
    at compile (file:///home/rickplumeria/tmp/remark-mdc-bug/node_modules/mdast-util-from-markdown/lib/index.js:337:40)
    at fromMarkdown (file:///home/rickplumeria/tmp/remark-mdc-bug/node_modules/mdast-util-from-markdown/lib/index.js:173:27)
    at parser (file:///home/rickplumeria/tmp/remark-mdc-bug/node_modules/remark-parse/lib/index.js:33:12)
    at Function.parse (file:///home/rickplumeria/tmp/remark-mdc-bug/node_modules/unified/lib/index.js:668:12)
    at parse (file:///home/rickplumeria/tmp/remark-mdc-bug/node_modules/unified-engine/lib/file-pipeline/parse.js:59:36)
    at wrapped (file:///home/rickplumeria/tmp/remark-mdc-bug/node_modules/trough/index.js:115:27)
    at next (file:///home/rickplumeria/tmp/remark-mdc-bug/node_modules/trough/index.js:65:23)
    at done (file:///home/rickplumeria/tmp/remark-mdc-bug/node_modules/trough/index.js:148:7)
    at file:///home/rickplumeria/tmp/remark-mdc-bug/node_modules/unified-engine/lib/file-pipeline/read.js:49:7

✖ 1 error

```

**Comment by abdonrd** [comment]
I have the same error using `remark-parse@11`
https://github.com/remarkjs/remark/releases/tag/remark-parse%4011.0.0

**Comment by farnabaz** [comment]
This should be fixed in v3.0.0.
